contracts/staking/Staking.sol

```diff
@@ -717,6 +717,14 @@ contract Staking is StakingV1Storage, GraphUpgradeable, IStaking {
         _withdraw(msg.sender);
     }
 
+    /**
+     * @dev Set the destination where to send rewards.
+     * @param _destination Rewards destination address. If set to zero, rewards will be restaked
```

Comment:
great idea that set to 0 = restaked. this way, when the upgrade happens, it will automatically be set so that everyone is re-staking. we won't run into any breaks